tests/reg_tests/convert_old_refs.py

```diff
@@ -0,0 +1,47 @@
+import glob
```

Comment:
I used the function to generate the new reference files. So it does work. It can break when things are printed out of order because of some MPI shenanigans, but that only happened once in the 18 adflow reg test.

This function should probably go in the centralized registration testing modules.